README.md

```diff
@@ -14,15 +14,15 @@ you will need to change the:
 play.http.secret.key = ...
 ``` 
 
-in `application.conf` to different value, as we will be using the `start` mode to launch the play app.
+in `application.conf` to different value, as we will be using the `runProd` mode to launch the play app.
 
 
 ## Running
 
 Run this using [sbt](http://www.scala-sbt.org/).  If you downloaded this project from http://www.playframework.com/download then you'll find a prepackaged version of sbt in the project directory:
 
 ```
-sbt start
+sbt runProd
```

Comment:
`start` is deprecated and logs a warning when you use it… `runProd` is the newer alternative for interactive sbt usage